chapters/dashboard/index.md

```diff
@@ -1,186 +1,21 @@
-# Using Dashboard to Monitor Logs and View Flow
+# Jina Dashboard
 
-**Dashboard** (beta) offers the insight of running tasks in Jina. With dashboard, one can analyze logs, design flows and view Jina Hub images.
+**Jina Dashboard** is a low-code monitoring and management environment for Jina. With the Dashboard you can build Jina Flows, manage them from a central location and get insights into the health of the Flow through log stream analysis!
 
-![flow demo](flow-demo.gif)
+Use Jina Dashboard by visiting the [website](https://dashboard.jina.ai) or cloning it from [GitHub](https://github.com/jina-ai/dashboard) 
 
-🌟 **Features:**
+![flow demo](overview.gif)
 
-- Log streaming, real-time chart on log-level.
-- Grouping logs by Pods, Executors. Full text search on logs.
-- Drag & drop flow design, setting properties of each Pod via a webform.
-- Flow can be imported from/exported to YAML.
+# Features
 
+-  ##  [Flow Design GUI:](flow.md)
```

Comment:
https://docs.google.com/document/d/1DVSUMRye5icmtwGCRBpwmvKgKtZOiczdVSH0NRg6xvQ/edit

This document can help making the dashboard and hub features descriptions more complete :)